mathesar_ui/src/systems/table-view/row/RowCell.svelte

```diff
@@ -58,6 +68,9 @@
   $: columnId = column.id;
   $: ({ activeCell, selectedCells } = selection);
   $: isActive = $activeCell && isCellActive($activeCell, row, processedColumn);
+  $: ({ pkColumn } = columnsDataStore);
+  $: primaryKeyColumnId = $pkColumn?.id;
+  $: rowKey = getRowKey(row, primaryKeyColumnId);
```

Comment:
The property `primaryKeyColumnId` can be passed down directly as a prop from `Row` to `RowCell` instead of taking it from the tabularData context, since Row already does that.

Reply:
I think it's done, you may check.